doc/userDocs/usersGuide.md

```diff
@@ -1039,34 +1039,34 @@ instances you will be able to deploy on your clusters.
+| 6                                     |  4\.75   |  33\.40           |  192\.00        |  15\.54  |  75\.06           |  354\.20        |  28\.80  |  191\.02          |  480\.00        |
+| 7                                     |  5\.54   |  38\.96           |  224\.00        |  18\.13  |  87\.57           |  413\.40        |  33\.60  |  222\.85          |  560\.00        |
+| 8                                     |  6\.33   |  44\.53           |  256\.00        |  20\.72  |  100\.08          |  472\.60        |  38\.40  |  254\.69          |  640\.00        |
+| 9                                     |  7\.12   |  50\.10           |  288\.00        |  23\.31  |  112\.59          |  531\.80        |  43\.20  |  286\.52          |  720\.00        |
+| 10                                    |  7\.91   |  55\.66           |  320\.00        |  25\.90  |  125\.10          |  591\.00        |  48\.00  |  318\.36          |  800\.00        |
+| 11                                    |  8\.70   |  61\.23           |  352\.00        |  28\.49  |  137\.61          |  649\.20        |  52\.80  |  350\.20          |  880\.00        |
+| 12                                    |  9\.49   |  66\.80           |  384\.00        |  31\.08  |  150\.12          |  708\.40        |  57\.60  |  382\.03          |  960\.00        |
+| 13                                    |  10\.28  |  72\.36           |  416\.00        |  33\.67  |  162\.63          |  767\.60        |  62\.40  |  413\.87          |  1,040\.00      |
+| 14                                    |  11\.07  |  77\.93           |  448\.00        |  36\.26  |  175\.14          |  826\.80        |  67\.20  |  445\.70          |  1,120\.00      |
+| 15                                    |  11\.87  |  83\.50           |  480\.00        |  38\.85  |  187\.65          |  886\.00        |  72\.00  |  477\.54          |  1,200\.00      |
+| 16                                    |  12\.66  |  89\.06           |  512\.00        |  41\.44  |  200\.16          |  944\.20        |  76\.80  |  509\.38          |  1,280\.00      |
+| 17                                    |  13\.45  |  94\.63           |  544\.00        |  44\.03  |  212\.67          |  1,003\.40      |  81\.60  |  541\.21          |  1,360\.00      |
+| 18                                    |  14\.24  |  100\.20          |  576\.00        |  46\.62  |  225\.18          |  1,062\.60      |  86\.40  |  573\.05          |  1,440\.00      |
+| 19                                    |  15\.03  |  105\.76          |  608\.00        |  49\.21  |  237\.69          |  1,121\.80      |  91\.20  |  604\.88          |  1,520\.00      |
+| 20                                    |  15\.82  |  111\.33          |  640\.00        |  51\.80  |  250\.20          |  1,181\.00      |  96\.00  |  636\.72          |  1,600\.00      |
```

Comment:
That is a lot of 0.2 multiples caused by rounding 0.15. (due to 2150).

Reply:
Yes.  The additional size is relatively small so I considered ignoring it, but I decided to err on the side of ensuring that we don't under-report the requirements.